docs/documentation/logging.rst

```diff
@@ -0,0 +1,61 @@
+.. _logging:
+
+Logging
+=======
+
+Logging in audiomate is done using the standard Python logging facilities.
+
+Enable Logging
+--------------
+
+By default, only messages of severity ``Warning`` or higher are printed to ``sys.stderr``.
+Audiomate provides detailed information about progress of long-running tasks with messages of severity ``Info``.
+In order to enable such messages for levels of lower severity, logging has to be configured.
```

Comment:
You can drop that sentence.

Reply:
Yes, seems obvious after you pointed it out :).